controllers/users-controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchUsers } = require("../models/users-models.js");
+
+exports.getUsers = (req, res, next) => {
+  return fetchUsers()
+    .then((user) => {
+      res.status(200).send({ users: user });
```

Comment:
please change the return of the .then to users. Is more syntactically correct and you don't need the {users: user}